src/cryptoadvance/specter/server_endpoints/settings.py

```diff
@@ -233,6 +234,87 @@ def general():
     )
 
 
+@settings_endpoint.route("/tor", methods=["GET", "POST"])
+@login_required
+def tor():
```

Comment:
Some documentation maybe something like:
'''
controls the tor-behaviour
POST only
param action might be "save", "test_tor" or "toggle_hidden_service"
param proxy_url usually something like socks5h://localhost:9050
param only_tor "on" or something else (off)
...
'''